packages/eas-cli/src/credentials/android/actions/new/DownloadKeystore.ts

```diff
@@ -0,0 +1,81 @@
+import chalk from 'chalk';
+import fs from 'fs-extra';
+
+import { AndroidAppBuildCredentialsFragment } from '../../../../graphql/generated';
+import Log from '../../../../log';
+import { confirmAsync } from '../../../../prompts';
+import { maybeRenameExistingFileAsync } from '../../../../utils/files';
+import { Context } from '../../../context';
+import { AppLookupParams, formatProjectFullName } from '../../api/GraphqlClient';
+import { displayAndroidKeystore } from '../../utils/printCredentialsBeta';
+
+interface DownloadKeystoreOptions {
+  app: AppLookupParams;
+  displaySensitiveInformation?: boolean;
+  outputPath?: string;
+}
+
+export class DownloadKeystore {
+  constructor(private options: DownloadKeystoreOptions) {}
+
+  public async runAsync(
+    ctx: Context,
+    buildCredentials: AndroidAppBuildCredentialsFragment
+  ): Promise<void> {
+    const keystore = buildCredentials.androidKeystore;
+    if (!keystore) {
+      Log.warn(
+        `There is no valid Keystore defined for build credentials: ${buildCredentials.name}`
+      );
+      return;
+    }
+
+    let displaySensitiveInformation = this.options?.displaySensitiveInformation;
+    if (displaySensitiveInformation === undefined && !ctx.nonInteractive) {
+      displaySensitiveInformation = await confirmAsync({
+        message: 'Do you want to display the sensitive information of the Android Keystore?',
+      });
+    }
+    const projectFullName = formatProjectFullName(this.options.app);
+    const keystorePath = this.options?.outputPath ?? `${projectFullName.replace('/', '__')}.jks`;
+
+    await maybeRenameExistingFileAsync(ctx.projectDir, keystorePath);
+
+    Log.log(`Saving Keystore to ${keystorePath}`);
+    await fs.writeFile(keystorePath, keystore.keystore, 'base64');
+
+    // non-sensitive information
+    displayAndroidKeystore(keystore);
+    Log.newLine();
+
+    if (displaySensitiveInformation) {
+      Log.log(`Sensitive Keystore information:
+    Keystore password: ${chalk.bold(keystore.keystorePassword)}
+    Key alias:         ${chalk.bold(keystore.keyAlias)}
+    Key password:      ${chalk.bold(keystore.keyPassword)}
+
+    Path to Keystore:  ${keystorePath}
+      `);
+    }
```

Comment:
```suggestion
    displayAndroidKeystore(keystore);
    if (displaySensitiveInformation) {
      Log.newLine();
      Log.log(`Sensitive Keystore information:
    Keystore password: ${chalk.bold(keystore.keystorePassword)}
    Key alias:         ${chalk.bold(keystore.keyAlias)}
    Key password:      ${chalk.bold(keystore.keyPassword)}

    Path to Keystore:  ${keystorePath}
      `);
    }
    Log.newLine();
```
Would it look better this way?